SYMBOLS_MANIFEST.txt

```diff
@@ -134,8 +134,10 @@ System`Automatic
 System`Axes
 System`Axis
 System`Background
+System`Backslash
```

Comment:
@rocky, the changes in the SYMBOLS_MANIFEST.txt are already done...

Reply:
Something I ran must have changed this. I definitely have not been changing this by hand. 

At any rate, just fix whatever problem you encounter.

Reply:
It is fixed automatically once you run the test locally, and the test passes. It does not seem to fail here. What is failing is the localization of the operator.json file, that is not generated in the place where mathics_scanner.tokenizer looks for it.